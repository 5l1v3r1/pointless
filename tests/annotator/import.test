
import "call.test" as foo
x = foo