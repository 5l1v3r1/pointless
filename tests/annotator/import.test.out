
x = foo
access: foo global: 0

import "call.test" as foo
entry: foo index: 0

x = foo
entry: x index: 1
